app/services/edition_unwithdrawer.rb

```diff
@@ -0,0 +1,44 @@
+class EditionUnwithdrawer < EditionService
+  def verb
+    'unwithdraw'
+  end
+
+  def past_participle
+    'unwithdrawn'
+  end
+
+  def failure_reason
+    @failure_reason ||= failure_reasons.first
+  end
+
+  def failure_reasons
+    reasons = []
+
+    reasons << "This action must be attributed to a user" if user.nil?
+    reasons << "An edition that is #{edition.current_state} cannot be #{past_participle}" unless can_transition?
+
+    reasons
+  end
+
+private
+
+  def prepare_edition
+    # The withdrawn edition have state 'published' to create a draft.
```

Comment:
The comment refers to [this code](https://github.com/alphagov/whitehall/blob/master/app/models/edition.rb#L409). It may be better to just change `Edition#create_draft` to work in the `withdrawn` state as well, but I'd need to look at the implications of this.